src/h5cpp/datatype/datatype.hpp

```diff
@@ -144,6 +145,20 @@ class DLL_EXPORT Datatype
   //!
   virtual void size(size_t size) const;
 
+  //!
+  //! @brief get sign of type in bytes
```

Comment:
I don't think this is accurate. It's definitely not "in bytes". I would maybe go with the description of the original C-API function in the HDF5 docs:
"Retrieves the sign type for an integer type."